src/lib/auth.ts

```diff
@@ -77,7 +77,7 @@ export const DEVICE_API_KEY_PERMISSIONS = [
 	`resin.device_tag.create?device/any(d:d/${matchesActor})`,
 	`resin.device_tag.update?device/any(d:d/${matchesActor})`,
 	'resin.application_config_variable.read?application/canAccess()',
-	'resin.release.read?should_be_running_on__device/canAccess() or belongs_to__application/canAccess()',
+	'resin.release.read?should_be_running_on__device/canAccess() or should_manage__device/canAccess() or belongs_to__application/canAccess()',
```

Comment:
do we _need_ it? I think the original permissions should allow this by the fact the supervisor application should be public.

Reply:
Is @richbayliss correct?  Can we revert this or should we leave it?

Reply:
i believe this is unnecessary, so i'm dropping it.